lib/sidebar.js

```diff
@@ -44,11 +44,13 @@ define(function () {
     };
 
     self.hide = function hide() {
-      container.classList.add('hidden');
+      container.children[1].classList.add('hide');
+      container.children[2].classList.add('hide');
     };
 
     self.reveal = function reveal() {
-      container.classList.remove('hidden');
+      container.children[1].classList.remove('hide');
+      container.children[2].classList.remove('hide');
```

Comment:
I think only other solution is a wrapper around tabs with content or add anther class to tab content and use the querySelector. Everything need to be changed if we adjust something.
I want to avoid a wrapper, because it could affect other selectors/functionally.